test/services/twilio-api/twilio-client.test.js

```diff
@@ -151,12 +151,29 @@ describe('services', () => {
         });
 
       test
-        .it('handles bad domains', async () => {
+        .it('handles bad paths', async () => {
           await expect(client.create({
             domain: 'api',
             path: '/2010-04-01/Accounts/{AccountSid}/AWESOME!!!!'
           })).to.be.rejectedWith('not found');
         });
+
+      test
+        .nock('https://api.twilio.com', api => {
+          api.post(`/2010-04-01/Accounts/${accountSid}/Addresses.json`).reply(201, {
+            verified: 'true'
+          });
+        })
+        .it('handles boolean parameters', async () => {
+          const response = await client.create({
+            domain: 'api',
+            path: '/2010-04-01/Accounts/{AccountSid}/Addresses.json',
+            data: { emergencyEnabled: true }
+          });
+
+          expect(response).to.eql({ verified: 'true' });
+          expect(httpClient.lastRequest.formData).to.eql({ EmergencyEnabled: 'true' });
```

Comment:
should `EmergencyEnabled` be `emergencyEnabled`?

Reply:
No, it gets PascalCased by the API client, which is what the API endpoint is expecting: https://www.twilio.com/docs/usage/api/address#create-an-address-resource